internal/upload/koji/koji.go

```diff
@@ -162,7 +162,7 @@ func NewFromPlain(server, user, password string, transport http.RoundTripper) (*
 	// The API doesn't require sessionID, sessionKey and callnum yet,
 	// so there's no need to use the custom Koji RoundTripper,
 	// let's just use the one that the called passed in.
-	rhTransport := &rh.RoundTripper{Client: rh.NewClient()}
```

Comment:
I guess this is only relevant in tests? I got a bit tripped up trying to figure out how it related to koji init / finalise.

Reply:
That's correct, I think it's important to have the code path tested to avoid deploying untested code.

Reply:
I agree, code looks good, just needs clarification in the commit message, I think.

Reply:
I think this is clear now from the context.